golden_response/staging/get_stats/NYTCovid19CumulativeCases.json

```diff
@@ -394,7 +468,44 @@
       "2020-05-28": 2701,
       "2020-05-29": 2707,
       "2020-05-30": 2731,
-      "2020-05-31": 2776
+      "2020-05-31": 2776,
+      "2020-06-01": 2805,
+      "2020-06-02": 2819,
+      "2020-06-03": 2832,
+      "2020-06-04": 2850,
+      "2020-06-05": 2892,
+      "2020-06-06": 2920,
+      "2020-06-07": 2942,
+      "2020-06-08": 2973,
+      "2020-06-09": 3017,
+      "2020-06-10": 3032,
+      "2020-06-11": 3063,
+      "2020-06-12": 3117,
+      "2020-06-13": 3157,
+      "2020-06-14": 3197,
+      "2020-06-15": 3230,
+      "2020-06-16": 3254,
+      "2020-06-17": 3288,
+      "2020-06-18": 3363,
+      "2020-06-19": 3398,
+      "2020-06-20": 3458,
+      "2020-06-21": 3547,
+      "2020-06-22": 3606,
+      "2020-06-23": 3727,
+      "2020-06-24": 3832,
+      "2020-06-25": 3888,
+      "2020-06-26": 3984,
+      "2020-06-27": 3984,
+      "2020-06-28": 4162,
+      "2020-06-29": 4162,
+      "2020-06-30": 4231,
+      "2020-07-01": 4572,
+      "2020-07-02": 4750,
+      "2020-07-03": 4849,
+      "2020-07-04": 5077,
+      "2020-07-05": 5273,
+      "2020-07-06": 5408,
+      "2020-07-07": 5478
```

Comment:
As I have use the new cache with branch cache capability. Can freeze the base cache for covid data. Then the data here should get pretty stable as well.
@tjann

Reply:
Ok, sounds good.